client/systems.go

```diff
@@ -103,3 +103,25 @@ func (client *Client) DoSystemAction(systemLabel string, action *SystemAction) e
 	}
 	return nil
 }
+
+// DoSystemReboot issues a request to reboot into the given label/mode
+func (client *Client) DoSystemReboot(systemLabel, mode string) error {
```

Comment:
Should this be called just "SystemReboot()"?

Reply:
Mabye RebootToSystem? also the doc comment needs to be expanded to explain what happens if label and/or mode are empty?